MobileWallet/UIElements/EmoticonView.swift

```diff
@@ -67,7 +67,24 @@ class EmoticonView: UIView {
     private var labelWidthConstraint: NSLayoutConstraint?
 
     private var emojiText: String!
-    private let RADIUS_POINTS: CGFloat = 6.0
+
+    private var RADIUS_POINTS: CGFloat = 6.0 {
```

Comment:
Change to lower case here, now that it's not a const anymore. Kutsal and I decided to also just not use any capital naming conventions going forward so if you see any please refactor.

Reply:
@SerhiiShovkoplias yeah please go through [this one](https://swift.org/documentation/api-design-guidelines).